web_widget_image_url/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
```

Comment:
usually copyright and licenses not needed in `__init__.py`

Reply:
Done